### PR TITLE
Add newtype wrappers

### LINKED ISSUE #4: Newtypes for strings
I think this is a good idea but don't have a good intuition for where to do this and where not too.

I imagine there could be things like:
- `ModuleName`
- `WebIDL`
- `PureScript`

and also possibly things like:
- `MemberName`
- `NodeName`
- `NodeTarget`
- `TypeElement`
- `TypeName`
- `TypeFamily`

but generally unsure.

I'm also not clear in which file(s) each of these should be defined.

@paf31 – what did you have in mind?

### PR DESCRIPTION
**Pull request by paulyoung**
This should close #4.

@paf31 I'd appreciate it if you could take a look and give any feedback 🙏

### CODE REVIEW

File: src/WebIDLFFI/Types.purs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  eq (PureScript x) (PureScript y) = x == y
+
+instance showPureScript :: Show PureScript where
+  show (PureScript x) = x
+
+prettyPrintPureScript :: PureScript -> String
+prettyPrintPureScript (PureScript x) = x
+
+
+newtype Module = Module String
```
Comment (paulyoung) [inline src/WebIDLFFI/Types.purs]: I think I should probably rename this  to `ModuleName`.

File: src/WebIDLFFI/Types.purs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+newtype PureScript = PureScript String
+
+instance eqPureScript :: Eq PureScript where
+  eq (PureScript x) (PureScript y) = x == y
+
+instance showPureScript :: Show PureScript where
+  show (PureScript x) = x
+
+prettyPrintPureScript :: PureScript -> String
+prettyPrintPureScript (PureScript x) = x
```
Comment (paulyoung) [inline src/WebIDLFFI/Types.purs]: I wasn't sure what to call this.

Comment (paulyoung) [inline src/WebIDLFFI/Types.purs]: Or even if there was a better way that didn't involve it.

File: src/WebIDLFFI/Types.purs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+module WebIDLFFI.Types
+  ( prettyPrintPureScript
+  , Module(Module)
```
Comment (paf31) [inline src/WebIDLFFI/Types.purs]: Maybe use the shorter `..` form here?

**Comment by paf31** [comment]
👍 Looks great!

**Pull request merged**